### ISSUE TITLE
https://regapp.mss.mghpcc.org/

### ISSUE THREAD

**Issue by hpdempsey** [nerc-project/operations#763]
User attempted to create a NERC account trhrough https://regapp.mss.mghpcc.org/ , using ORCID authentication provider.   This provider is listed in the "?" info, but is not in the dropdown list.   User was unable to login successfully.  "I tried with ORCID, and after granting the system access to my ORCID account and logging in with ORCID, it gives me a 500 Error."   User is dean.elzinga@ibm.com.  This access is supporting theAI4DD project, which is a high priority.   Please investigate ASAP and let us know whether ORCID is actually a supported IDP or not.

**Comment by Milstein** [comment]
@hpdempsey : Can @ibm.com use other OAuth instead of ORCID?

**Comment by joachimweyl** [comment]
@Milstein is ORCID broken?

**Comment by Milstein** [comment]
Yes, ORCID does not return all fields that is required by our RegApp. But IBM folks can use other authentication if that works i.e. Google or MS?

**Comment by Milstein** [comment]
@hpdempsey @joachimweyl : I can meet with the person who has this issue to fix it if ORCID can be used by some tweaks on their ORCID profile.

![Image](https://github.com/user-attachments/assets/9cde7961-723f-45a4-b037-f3b4a4336198)

**Comment by joachimweyl** [comment]
@Milstein do we want to support ORCID if so then we need to document steps for supporting ORCID?
